agent/03_agent_deployment.sh

```diff
@@ -13,6 +13,18 @@ source $SCRIPTDIR/agent/common.sh
 
 early_deploy_validation
 
+function add_host_dns_entries() {
+    ip_list=$1
+    num_ips=${#ip_list[@]}
+
+    for (( i=0; i<$num_ips; i++ ))
+    do
+        hostname=${FLEETING_HOSTNAME}${i}
```

Comment:
Especially if in future we're going to add more entries for the other nodes, then I guess it could be better to be coherent with the current dev-scripts naming conventions and use `MASTER_HOSTNAME_FORMAT` and `WORKER_HOSTNAME_FORMAT`

Reply:
OK, changed to use the MASTER_HOSTNAME_FORMAT and WORKER_HOSTNAME_FORMAT instead of redefining it